packages/flutter_tools/lib/src/commands/test.dart

```diff
@@ -131,6 +132,14 @@ class TestCommand extends FlutterCommand with DeviceBasedDevelopmentArtifacts {
         defaultsTo: 'coverage/lcov.info',
         help: 'Where to store coverage information (if coverage is enabled).',
       )
+      ..addMultiOption('coverage-package',
+        help: 'A regular expression matching packages names '
+              'to include in the coverage report (if coverage is enabled). '
+              'Current package is always included. '
+              'By default, the coverage will be collected only for the current package.',
+        valueHelp: 'package-name-regexp',
+        splitCommas: false,
```

Comment:
why no split commas?

Reply:
@p-mazhnik do we have to disallow split commas?

Reply:
If we use regex, we should, because comma can be a part of the regular expression

Reply:
Ahh yeah, I thought you could escape these with quotes from the command line, but I guess not. Good call.